datev_export_xml/views/templates.xml

```diff
@@ -85,15 +85,17 @@
                 t-att-invoice_id="doc.datev_invoice_id()"
                 t-att-invoice_type="doc.datev_invoice_type()"
             />
-
             <invoice_party t-call="datev_export_xml.export_party">
-                <t
-                    t-set="partner"
-                    t-value="doc.partner_id"
-                    t-if="doc.move_type in ['out_invoice', 'out_refund']"
-                />
-                <t t-set="partner" t-value="doc.company_id.partner_id" t-else="" />
-                <t t-set="account" t-value="partner.property_account_receivable_id" />
+                <t t-if="doc.move_type in ['out_invoice', 'out_refund']">
+                    <t t-set="partner" t-value="doc.partner_id" />
+                    <t
+                        t-set="account"
+                        t-value="partner.property_account_receivable_id"
+                    />
+                </t>
+                <t t-else="">
+                    <t t-set="partner" t-value="doc.company_id.partner_id" />
+                </t>
```

Comment:
It's an actual change now. Previously account was always set to the receivable of partner. Now it's not in the `t-else` branch. Why do you change the tested logic here when the previous code worked?

Do you have an actual problematic case? Do you fine it hard to read with the `t-set t-value t-if` in one tag?